### ISSUE TITLE
Why it requires redis < 3?

### ISSUE THREAD

**Issue by turicas** [webrecorder/pywb#817]
The [current redis Python package version is 4.5.1](https://pypi.org/project/redis/) - I'm using this version in a project that also requires pywb, but pywb requires `redis < 3`. Is there any specific reason to have this old version required?

**Comment by ikreymer** [comment]
No, just haven't had time to migrate to the latest. I believe there were significant API changes in the client.
PRs welcome!
